admin/admin.php

```diff
@@ -121,4 +172,27 @@ public function add_filters() {
 			$this->$obj = new $class( $this );
 		}
 	}
+	/**
+	 * Retrieve the locale according to the current language instead of the language
+	 * of the admin interface.
+	 *
+	 * @since 2.9
+	 *
+	 * @return string
+	 */
+	public function get_locale() {
```

Comment:
I would rename this method now to something like `get_locale_for sanitization` or something like that to be more precise on its usage.

Reply:
Done